sql/core/v2.3/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcFilterSuite.scala

```diff
@@ -44,6 +46,11 @@ import org.apache.spark.sql.types._
  */
 class OrcFilterSuite extends OrcTest with SharedSparkSession {
 
+  override protected def sparkConf: SparkConf =
```

Comment:
shall we do the same for the one in  v1.2?

Reply:
+1 for @cloud-fan 's comment.

Reply:
Addressed for v1.2. Thanks!

Reply:
Please update the description accordingly.